src/form-data/ReviewPage.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useEffect, useLayoutEffect } from 'react';
+import {
+  FormikBag,
+  FormikComputedProps,
+  FormikContextType,
+  FormikProps,
+  useFormikContext,
+} from 'formik';
+import { Link, useLocation } from 'react-router-dom';
+import { VaOnThisPage } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { Page, PageProps } from '../routing';
+import { FieldObject, PageObject } from './types';
+
+import { parseDate } from '../utils/helpers';
+import { PageContext } from './PageContext';
+// import { getRadioLabel } from "../utils/helpers";
+
+/**
+ * Transforms fields value into value that is more readable
+ * @param {boolean | object | string} field object value
+ * @param {string} key object key name
+ *
+ * @beta
+ */
+const transformFieldValue = (key: number, field: FieldObject) => {
+  if (typeof field?.value === 'undefined' || typeof field?.value === 'object')
+    return;
+
+  if (field?.options) {
+    const fieldIndex = field.options.findIndex(
+      (fieldOption) => fieldOption.value === field?.value
+    );
+    if (fieldIndex > 0) {
+      return field.options[fieldIndex].label;
+    }
+  }
+  if (field.value === 'true' || field.value === true) {
+    return 'Yes';
+  }
+  if (field.value === 'false' || field.value === false) {
+    return 'No';
```

Comment:
It's a nitpick, but could we store `Yes` and `No` in constants outside the function body instead of hard coding them here?